sematic/types/types/union.py

```diff
@@ -88,3 +89,8 @@ def _get_value_type(value: Any, type_: Any) -> Any:
             return unioned_type
 
     return type(value)
+
+def is_union(type_: Any) -> bool:
+    origin_type = get_origin_type(type_)
+
+    return origin_type==Union
```

Comment:
I want to know if this equality check is fine with everyone. The other choice would be to use a `can_cast` here between the input and union type. But based on my understanding, the `can_cast` function checks if we can cast the type to any of the types specified inside `Union` and not the `Union` type itself. Do correct me if I am wrong here.

Reply:
I think this works, also `is` works here.

Also, add spaces around the operator:
```python
origin_type is Union
```

Reply:
Sure I will update it.